dts/bindings/ethernet/phy_microchip_ksz8081.yaml

```diff
@@ -0,0 +1,28 @@
+# Copyright 2023 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+description: Microchip KSZ8081 Ethernet PHY device
+
+compatible: "microchip,ksz8081"
+
+include: ethernet-phy.yaml
+
+properties:
+  mc,interface:
```

Comment:
Should this property reside in `ethernet-phy.yaml`?

Reply:
yes, this phy can be configured for either mii or rmii interface

Reply:
I think I misread this comment, I think I know what you meant now, maybe this property can be generic and used by the enet driver since it has a phandle reference to the phy